lightning-invoice/Cargo.toml

```diff
@@ -10,11 +10,19 @@ readme = "README.md"
 
 [dependencies]
 bech32 = "0.7"
+bitcoin = "0.26"
 lightning = { version = "0.0.13", path = "../lightning" }
 secp256k1 = { version = "0.20", features = ["recovery"] }
 num-traits = "0.2.8"
 bitcoin_hashes = "0.9.4"
 
+[dev-dependencies.bitcoin]
+version = "0.26"
+features = ["bitcoinconsensus"]
```

Comment:
Not that it matters, but I think the solution here is to make rust-lightning's _test_utils feature depend on "bitcoin/bitcoinconsensus" instead.